### ISSUE TITLE
[MIGRATION] - Nord

### ISSUE THREAD

**Issue by Mouzzi-C** [isogeo/migrations-toolbelt#169]
# Prestation de Migration - *Nord*

Groupe cible : [12b69e6cde5546c9b2b32883db93ef99](https://app.isogeo.com/groups/12b69e6cde5546c9b2b32883db93ef99/inventory/search?p=1&ob=%23relevance&od=des)

Deadline : Pas de deadline précise

Contexte : Projet relancé en 2021. Fiches renseignée manuellement à migrer car le client n'avez pas le scan initialement. 

## Catalogues

- [ ] les catalogues Source et Cible existent:
  * *Nom du catalogue source* : [*uuid du catalogue source*](*lien vers le catalogue source dans app*)
  * *Nom du catalogue cible* : [33a7326080e24e539b2131a902ab517d](https://app.isogeo.com/groups/12b69e6cde5546c9b2b32883db93ef99/inventory/search?q=catalog:33a7326080e24e539b2131a902ab517d)
  * *M_Migrated* : [ff3bb01aac1c4fec8904bf92b3775f7a](https://app.isogeo.com/groups/12b69e6cde5546c9b2b32883db93ef99/inventory/search?p=1&ob=%23relevance&od=des&q=catalog%3Aff3bb01aac1c4fec8904bf92b3775f7a)

## Critères de correspondance

La cliente a pris le temps d'ajouter manuellement le nom de la donnée (sans le schéma) au titre des fiches sources afin que l'on puisse établir une correspondance. 

exemple : 

Fiche source : [Evaluation 2019 des bibliothèques selon la typologie ABD (cd59_diaglp_typoabd)](https://app.isogeo.com/groups/12b69e6cde5546c9b2b32883db93ef99/resources/f24df016074f4e96af0263efa30c7cd7/identification)

Fiche cible : [culture.cd59_diaglp_typoabd](https://app.isogeo.com/groups/12b69e6cde5546c9b2b32883db93ef99/resources/a4c7a23fb9c347dbab700c8d0f2ba7f0/history)

## Règles de migration (si nécessaire)

Migrer la documentation de la fiche source si une fiche cible est documentée. 

Migrer les événements. (événements tous créés manuellement)

## Suivi

- [ ] table de correspondance : *en attente du client*
- [ ] migration : *en attente du scan*
- [ ] suppression des fiches source : *en attente du client*

**Comment by SimonSAMPERE** [comment]
@Mouzzi-C , dans le cas de certaines fiches source, la client à écrit plusieurs valeurs séparées par des "/" entre parenthèses dans le titre. Est-ce que ça veut dire qu'il faut migrer la source dans plusieurs fiches cible ?

**Comment by SimonSAMPERE** [comment]
Bon je suis parti du principe que oui et du coup voici une [première table de correspondance](https://github.com/isogeo/migrations-toolbelt/files/7846930/correspondances_11-01-2022-14h24.csv), il reste 227 matches à établir.

**Comment by Mouzzi-C** [comment]
yep, j'étais justement en train de te checker ça. t'as bien fait 👍
